docs/semgrep-cloud-platform/bitbucket-pr-comments.md

```diff
@@ -0,0 +1,163 @@
+---
+slug: bitbucket-pr-comments 
+append_help_link: true
+title: BitBucket PR comments 
+hide_title: true
+description: "Enable PR comments in your BitBucket repositories to display Semgrep findings to developers."
```

Comment:
```suggestion
description: "Enable PR comments in your Bitbucket repositories to display Semgrep findings to developers."
```

There are a number of cases where `BitBucket` should be `Bitbucket` and rather than suggesting a change for each, I’ll let you do a find-and-replace which should be much faster.

Reply:
Yeah, I should have set up my snippets to autocorrect that in my environment. My bad.

Reply:
<img width="508" alt="Snag_1c9189a" src="https://user-images.githubusercontent.com/20766840/234461099-b1bd3e39-51a3-4ff8-98c5-75eac0d6a0e5.png">

Running Vale so it doesn't happen again.